docs/upgrade/upgrade-security.asciidoc

```diff
@@ -11,13 +11,13 @@ NOTE: For detailed information about the 8.0 release, check out the <<release-no
 
 [discrete]
 [[upgrade-8.1.1]]
-== Upgrade to 8.1.1
+== Avoid upgrading to 8.1.1
 
 IMPORTANT: There is a known issue that significantly impacts UI responsiveness. Therefore, we recommend to skip upgrading to this version.
 
 [discrete]
 [[upgrade-reqs]]
-== Upgrade to an earlier 8.x version
+== Upgrade to other 8.x versions
```

Comment:
Revised because this was saying "8.x versions earlier than 8.1.1" (based on the previous section "Upgrade to 8.1.1."), but I think this info also applies to 8.x versions _later_ than 8.1.1, right? Basically any 8.x version that _isn't_ 8.1.1.